File: Java/transaction-management/src/main/java/com/dwivedi/transaction/management/models/Account.java
package com.dwivedi.transaction.management.models;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;

import java.util.UUID;

@Entity
public class Account {
    @Id
    @GeneratedValue(strategy = GenerationType.AUTO)
    private UUID id;
    private double balance;

    public UUID getId() {
        return id;
    }

    public void setId(UUID id) {
        this.id = id;
    }

    public double getBalance() {
        return balance;
    }

    public void setBalance(double balance) {
        this.balance = balance;
    }
}

File: Kotlin/transaction-management-api/src/main/kotlin/com/dwivedi/transaction/management/api/models/Account.kt
package com.dwivedi.transaction.management.api.models

import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id
import java.util.UUID

@Entity
data class Account(
        @Id @GeneratedValue(strategy = GenerationType.AUTO) val id: UUID,
        val balance: Double,
)
